src/components/Cards.js

```diff
@@ -43,8 +73,8 @@ export default class Card {
     this._cardElement.querySelector(".card__image").src = this.link;
     this._cardElement.querySelector(".card__image").alt = this.name;
     this._cardElement.querySelector(".card__title").textContent = this.name;
-
-    this._setEventListeners();
+    this.setButtonState();
```

Comment:
`this.setButtonState();` should be called **after** `setEventListeners` to be able to use `this._likeButton` inside it


```js

    this.setEventListeners();
    this.setButtonState();
```


So, you'll have this code in `setButtonState`


```js

  setButtonState() {
    if (this._isLiked) {
      this._likeButton.classList.add("card__like-button_active");
    } else {
      this._likeButton.classList.remove("card__like-button_active");
    }
  }

```